javascripts/discourse/initializers/tag-icons.js.es6

```diff
@@ -73,8 +73,29 @@ export default {
   name: "tag-icons",
 
   initialize() {
-    withPluginApi("0.8.31", (api) => {
+    withPluginApi("1.6.0", (api) => {
       api.replaceTagRenderer(iconTagRenderer);
+
+      if (api.registerCustomTagSectionLinkPrefixIcon) {
+        const tagIconList = settings.tag_icon_list.split("|");
```

Comment:
I don't know the implementation, but you need to be sure `tag_icon_list`, can't be an empty string.

```javascript
"".split("|") // ['']
```

Reply:
This is ok because we have another check later that ensures each segment of the string is the right format before processing it.